style.css

```diff
@@ -0,0 +1,133 @@
+@font-face {
+    font-family: 'NeueMontrealBook';
+    src: url('PPNeueMontreal-Book.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealItalic';
+    src: url('PPNeueMontreal-Italic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealMedium';
+    src: url('PPNeueMontreal-Medium.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealSemiBoldItalic';
+    src: url('PPNeueMontreal-SemiBolditalic.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'NeueMontrealThin';
+    src: url('PPNeueMontreal-Thin.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'GoshaSansBold';
+    src: url('PPGoshaSans-Bold.otf') format('opentype');
+    font-weight: 900;
+    font-style: normal;
+}
```

Comment:
You have so many font faces here, but it seems like this is the only one you're actually using! So please remove the ones that aren't needed (along with the files)